### DISCUSSION TITLE
Camera Plugin - iOS Photo Library limited access

### DISCUSSION THREAD

**Discussion by martiBpl** [ionic-team/capacitor#5548] [Q&A]
Hey,

First of all I want to thank you for the wonderful job that you did in all Capacitor plugins.

I'm struggling with the Camera plugin's iOS 14 photo library permissions. When the Camera.getPhoto() is invoked at first the user has an option to allow only limited access to his private photo library.  But no mater how many photos he selects we are able to see all photos from the albums, moreover every picture can be selected and used in the getPhoto process. 

Is that a bug ? For sure it's a bad UX to display user privacy options that we'll not respect.

**Comment by c3d4ic** [comment]
Up,
It's not working even for iOS 14 and iOS 15+
When using pickLimitedLibraryPhotos, on iOS 15+ first times it will display the library view to select photos. After the selection and click on "done" we are expecting a limited view with only the selected photos.

**Comment by robingenz** [comment]
This has already been answered here: https://github.com/ionic-team/capacitor-plugins/issues/1325#issuecomment-1346182920
